githubflow_release/release.py

```diff
@@ -315,7 +313,7 @@ def release(defaults_file='gitflow_release.yml',
             base_branch='master',
             generate_debian_changelog=False,
             hotfix_pr_ids=None,
-            excluded_pr_tag=['hotfix', 'not_in_changelog']):
```

Comment:
don't you want to keep those defaults ?

Reply:
Yes, the default values are kept here:   
https://github.com/CanalTP/githubflow_release/pull/8/files/26e42932a9d20782be924a196b9116fe48c2840e#diff-8b0f9aa386d446f10d61c96a52d73366R30

(I must admit it's not explicit)

Reply:
here: https://github.com/CanalTP/githubflow_release/pull/8/files#diff-9d2b98b0efad08c014a5b58ad0c1b525R335